src/realm.h

```diff
@@ -11,6 +24,10 @@
 #include <stdbool.h>
 #include <realm/error_codes.h>
 
+#if !defined(REALM_APP_SERVICES)
+#include <realm/util/config.h>
+#endif
+
```

Comment:
Why do we need to include `config.h` here? It seems the only place this file is included in `features.h` prior to these changes.

Reply:
Building the RealmFFI target failed because REALM_APP_SERVICES wasn't defined in this header file, but it was included in the .cpp file (through indirect includes that pulled in features.h). config.h is where this was defined directly, so I included it here (there are only macros there which is fine for a C header). But I see now that it would be more clean to have this defined as part of the CMake compile definitions on the object-store target so I changed it to that instead. Thanks for pointing it out.